rule_packages/c/IO4.json

```diff
@@ -68,4 +68,4 @@
       "title": "Use valid format strings"
     }
   }
-}
```

Comment:
Revertable change

Reply:
This is because I automatically added the tags, rewriting these files.

I think removing the trailing newline is consistent with the other rule package files, so I will keep the change.